Analysis/python/variable_check.py

```diff
@@ -6,34 +6,30 @@
 parser.add_argument('--tree', required=True, type=str, help="Tree name")
 parser.add_argument('--max-entries', required=False, type=int, default=None,
                     help="Maximal number of entries to process")
+parser.add_argument('--std-thr', required=False, type=float, default=1e-7,
+                    help="If std(var) < std_thr, var is considered constant.")
```

Comment:
Asking out of my curiosity: do I understand correctly that then you will not consider these variables for the training at all?

Reply:
yes, this script was used to prune const variables before the production, so they are not included in the PR and will not be used for the training.